src/lua/init.c

```diff
@@ -1102,6 +1102,7 @@ run_script_f(va_list ap)
 	bool bytecode = opt_mask & O_BYTECODE;
 	bool debugging = opt_mask & O_DEBUGGING;
 	bool help_env_list = opt_mask & O_HELP_ENV_LIST;
+	bool failover = opt_mask & O_FAILOVER;
```

Comment:
Error in commit message:
> The failover **cooordinator** code is part of Tarantool Enterprise Edition.

How about squash this and the next patch? Also, we can add test for calling --failover in CE here.

Reply:
Fixed the typo.

Squashed the commits.

Added a simple test.

```diff
diff --git a/test/config-luatest/cli_test.lua b/test/config-luatest/cli_test.lua
index 25e311a300..116bdadbd3 100644
--- a/test/config-luatest/cli_test.lua
+++ b/test/config-luatest/cli_test.lua
@@ -131,3 +131,11 @@ g.test_force_recovery = function()
     t.assert_equals(res.exit_code, 0)
     t.assert_equals(res.stdout, 'false')
 end
+
+g.test_failover = function()
+    local opts = {nojson = true, stderr = true}
+    local res = justrun.tarantool('.', {}, {'--failover'}, opts)
+    t.assert_not_equals(res.exit_code, 0)
+    t.assert_equals(res.stderr, '--failover CLI option is available only in ' ..
+        'Tarantool Enterprise Edition')
+end
```